cloud-functions/estimate_probability_of_change/main.py

```diff
@@ -8,6 +8,9 @@
 REQUEST_KEY_NAME = "name"
 REQUEST_KEY_DURATION_IN_DAYS = "days"
 REQUEST_KEY_PERCENT_CHANGE = "percent_change"
+REQUEST_KEY_DURATION_IN_DAYS_LIST = "days_list"
```

Comment:
Instead of representing request like:

{
  "days_list": [...],
  "percent_change_list": [...]
}

I think we should have something like:
{
   "requests": [
      {
          "days": X,
          "percent_change": Y
      },
     {...}, ...
   ]
}

reason, in proposed schema len(days_list) might not be equal to len(percent_change_list) so one more check required. And if we will add more items to the request objects in the future we will have to have more such *_listfields. For example, do we need name_list?

Reply:
`so one more check required` this check is already part of code in function `calculate_batch_likelyhood`